### PR TITLE
Follow-up to 2384

### PR DESCRIPTION
**Pull request by kghbln**
This PR is made in reference to: #2384 

This PR addresses or contains:
- Made message strings translatable for notices and removals

This PR includes:
- [ ] Tests (unit/integration)
- [ ] CI build passed

### CODE REVIEW

File: includes/Settings.php [hunk 1]
```diff
@@ -306,15 +306,17 @@ private static function initLegacyMapping( &$configuration ) {
 		// output notices
 		$GLOBALS['smwgDeprecationNotices'] = array(
 			'notice' => array(
-				'smwgAdminRefreshStore' => 'deprecated in 2.5, to be removed in 3.1'
+				'smwgAdminRefreshStore' => '3.1.0',
+				'smwgQueryDependencyPropertyExemptionlist' => '3.1.0',
+				'smwgQueryDependencyAffiliatePropertyDetectionlist' => '3.1.0'
```
Comment (kghbln) [inline includes/Settings.php]: I added these to to provide a perspective on when they will be removed. To remove at the same time as the first setting listed seemed ok to me.

**Comment by kghbln** [comment]
@mwjames I hope that this works out and what you are ok with this. I think just adding version numbers to "Settings.php" is better.

I accidentally forgot to add [stop ci] to the last in the series of commits and I do not know how to stop it afterwards :(

This should be a squash and merge if acceptable.

**Comment by kghbln** [comment]
> I accidentally forgot to add [stop ci] to the last in the series of commits and I do not know how to stop it afterwards :(

Great. I did a trivial change to be able to add [skip ci] to learn that only the preceding commit is interrupted.

**Comment by JeroenDeDauw** [comment]
+1

Cool feature btw!

**Comment by mwjames** [comment]
Back-ported to 2.5.x with 76691c8.

**Comment by kghbln** [comment]
> Back-ported to 2.5.x with 76691c8.

Thanks a lot!

**Pull request merged**